### ISSUE TITLE
Missing HATU boundary

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#178]
Harriet Tubman Underground Railroad National Monument: http://www.nps.gov/hatu/index.htm.

Currently no boundary available.

**Comment by chadlawlis** [comment]
This boundary is not available in the latest Lands release, so we will have to reach out to the park/region.

**Comment by chadlawlis** [comment]
I emailed Nigel, who will send a boundary along next week.

**Comment by chadlawlis** [comment]
From Nigel, 10/19/15:

> Turns out the only HATU boundary I have is not suitable for public viewing, per the Lands Office.  I sent a message to Mark Duffy asking if he has anything releasable and will let you know what he has to say about that - whenever he says it.

**Comment by nateirwin** [comment]
It might still be nice to get Nigel's version of the boundary so we have *something to look at.

**Comment by chadlawlis** [comment]
Let's wait on it for now - last time she checked in with Mark he was quick to respond with an interim WEFA boundary. If we don't hear back by the end of the week I can ask if we can use it.

Are we including a disclaimer on Places that park management boundaries are not necessarily authoritative? That might help with some of these cases.

**Comment by nateirwin** [comment]
All of our maps have a link to our disclaimer page in the bottom right-hand corner: http://www.nps.gov/npmap/disclaimer/. I actually have updated language for this page from the Lands division, and I'll try to get it out shortly. But basically, yes, these boundaries are not authoritative. From the page:

> These are not engineering quality drawings, and should be used for administrative purposes only.

I do think we need to prioritize updating the language on that page so we make sure it is clear the boundaries we show in Park Tiles may not even be from the Lands Division.

**Comment by nateirwin** [comment]
New issue created here to track updating the text on the disclaimer page: https://github.com/nationalparkservice/npmap-website/issues/88.

**Comment by chadlawlis** [comment]
``` sql
insert into parks (unit_id, unit_name_short, unit_name_long, unit_desig_abbr, unit_desig_full, unit_code, unit_region, unit_state, unit_url) values ((select max(unit_id) + 1 from parks), 'Harriet Tubman Underground Railroad', 'Harriet Tubman Underground Railroad', 'NM', 'National Monument', 'hatu', 'ner', 'md', 'http://www.nps.gov/hatu/index.htm');
insert into parks_poly (unit_id, unit_code) values ((select unit_id from parks where unit_code = 'hatu'), 'hatu');
insert into parks_line (unit_id, unit_code) values ((select unit_id from parks where unit_code = 'hatu'), 'hatu');
```

**Comment by chadlawlis** [comment]
So, thanks to the NPMap.js' geocoder we can add a point for this based on the NPS.gov address provided: http://www.nps.gov/hatu/contacts.htm

2145 Key Wallace Drive
Cambridge, MD 21613

Hopped into http://www.nps.gov/lib/npmap.js/3.0.7/examples/geocoder-control.html, searched for the address, and ran `NPMap.config.L.getCenter();` in Chrome's console at landing zoom/extent, providing:

`L.LatLng {lat: 38.44700133282411, lng: -76.12298011779785}`

Plugged this into an update query and we should be good to go with a label point:

``` sql
UPDATE parks
SET geom_point = (SELECT ST_GeomFromText('POINT(38.44700133282411 -76.12298011779785)', 3857))
WHERE unit_code = 'hatu';
```

We still need a `geom_poly` for this, though.

**Comment by chadlawlis** [comment]
@mtaylorlong pointed out that the `geom_point` is showing up on Null Island.

We realized that when adding these coordinates we need to do so in 4326, then convert to 3857. I also realized that `ST_GeomFromText()` requires lon lat, not lat lon. So, to fix:

``` sql
-- reset point coordinate in 4326
UPDATE parks
SET geom_point = (SELECT ST_GeomFromText('POINT(-76.12298011779785 38.44700133282411)', 4326))
WHERE unit_code = 'hatu';

-- convert to 3857
UPDATE parks
SET geom_point = ST_Transform(ST_SetSRID(geom_point, 4326), 3857)
WHERE unit_code = 'hatu';
```

**Comment by chadlawlis** [comment]
From Nigel, 3/2/16:

> Mark has some unpublished boundary data and aims to work on it (between dungeon visits) to get it ready to pass to Ft. Collins asap.

**Comment by chadlawlis** [comment]
From Nigel, 3/9/16:

> Please remove COLT and HATU entirely from NPPlaces and NPMap.  Per the appended message from[Lands], COLT is not an NPS unit at this time.  He said there is a similar situation with HATU and to refrain from including that in anything we release from the NER as well.  It sounds like both are pending, but they have to stay 'dark' until the criteria in the enabling legislation is met and their status is officially changed.

See https://github.com/nationalparkservice/places-data/issues/182#issuecomment-194422651 for more context re: COLT.

So, we will hold off on adding a boundary for HATU until it is truly an "official" unit. We currently have a label, which I will remove.

``` sql
UPDATE
    parks
SET
    geom_point = NULL
WHERE
    unit_code = 'hatu';
```

Use code from above https://github.com/nationalparkservice/places-data/issues/178#issuecomment-178723513 to set the geom_point again, if appropriate to use a label before a boundary area becomes available.

**Comment by chadlawlis** [comment]
Still not official as of latest Lands release 2016063 (confirmed by Nigel @ NER).

**Comment by chadlawlis** [comment]
A "temporary" boundary was included in the 20170630 Lands release
